scripts/create_test_users.py

```diff
@@ -10,28 +10,61 @@
 SOURCE = 'script' + datetime.datetime.now().strftime('%Y%m%d.%H%M%S')
 
 NUM_WEEKS = 14
-PUA_FULL_TIME = 'PUA full-time'
-UI_FULL_TIME = 'UI full-time'
-UI_PART_TIME = 'UI part-time'
+PUA_FULL_TIME = 'PUA full time'
```

Comment:
good catch, I didn't notice this!

Reply:
I was so confused yesterday when none of the questions were showing up for the test users I created!

Reply:
Ah, I saw that too yesterday! Thanks for debugging and fixing.